component/otelcol/connector/grafanacloud/connector.go

```diff
@@ -0,0 +1,120 @@
+package grafanacloud
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/connector"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.uber.org/zap"
+)
+
+const (
+	hostInfoMetric     = "host_info"
```

Comment:
shall we call this `trace_host_info`? rationale: we may want to keep separate metrics for each of the telemetry signals (so far we only look at traces to extract this information).